.github/workflows/tpu_ci.yml

```diff
@@ -0,0 +1,33 @@
+name: TPU Integration Test
+run-name: TPU Testing
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 16,20,0 * * 1-5'
+jobs:
+  tpu-test:
+    runs-on: v4-runner-set
+    steps:
+      - name: Checkout and Setup PyTorch Repo
+        run: |
+          git clone --recursive https://github.com/pytorch/pytorch
+          cd pytorch/
+          python3 setup.py install --user
+      - name: Checkout PyTorch/XLA Repo
+        uses: actions/checkout@v4
+        with:
+          path: pytorch/xla
+      - name: Run PyTorch/XLA Setup
+        env:
+          BAZEL_VERBOSE: 1
+          BUNDLE_LIBTPU: 1
+          TPUVM_MODE: 1
+        run: |
+          cd pytorch/xla
+          python3 setup.py install --user
+      - name: Run Tests
+        env:
+          PJRT_DEVICE: TPU
+        run: |
+          cd pytorch/xla
+          python3 -u test/test_operations.py -v
```

Comment:
Can you add the rest of the tests from [here](https://github.com/pytorch/xla/blob/d6cabc5a45944365c0f3e6969ac4083eb425a5a3/test/tpu/xla_test_job.yaml#L47-L59), or move those tests into a script and invoke it from both places?

Reply:
Added